client/layout/masterbar/masterbar-notifications/notifications-button.jsx

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import { Component } from 'react';
 import { connect } from 'react-redux';
 import TranslatableString from 'calypso/components/translatable/proptype';
+import { BellIcon } from 'calypso/layout/global-sidebar/menu-items/notifications/icon';
```

Comment:
I'm kinda surprised that `@wordpress/icons` does not have a bell icon
https://wordpress.github.io/gutenberg/?path=/docs/icons-icon--docs